### ISSUE TITLE
MHR not rendering to Quest 2 via 6800XT

### ISSUE THREAD

**Issue by iiStryker** [praydog/REFramework#238]
Getting audio only
I've tried all the known fixes. Using the most recent Appveyor Dinput8.dll file.

AFAIK HDR is OFF. I have it turned off globally in Windows and have this as launch option in Steam +mat_hdr_level 0 (tried with and without launch option)

Tried unplugging my monitor HDMI and while that worked for RE7 & 8 it's not fixing the issue in MHR.

Clean install/ reinstall/and file integrity verified
Radeon driver 22.1.1
OpenXR runtime (i've tried both Oculus and OpenXR runtime)
Steam overlay and theater mode OFF
msconfig non-essential services disabled. no background apps running
Adrenalin GFX and display settings set to off or let App decide
game compatibility set to disable fullscreen optimizations/let app decide scaling

occasionally getting D3D fatal error (22) (stable in pancake mode)

[re2_framework_log.txt](https://github.com/praydog/REFramework/files/7945485/re2_framework_log.txt)

**Comment by cursey** [comment]
> Using the most recent Appveyor Dinput8.dll file.

Just to be clear, you grabbed one from within the last 16 hours right? There was a recent commit to address the HDR problem but from your log it looks like the game is still running with an HDR back buffer.

**Comment by iiStryker** [comment]
Yeah. Here's a new one just in case. I renamed the OG file installed the new Dinput8.dll. 

I have tried overwriting it as well to no avail.

[re2_framework_log.txt](https://github.com/praydog/REFramework/files/7945846/re2_framework_log.txt)

**Comment by ToeKnee02** [comment]
I was having this exact same issue before and was able to fix it. Below was the response from the mod creator.

> According to the log, you have HDR enabled. I know you said you don't have an HDR monitor in another issue, but this is the reason why it's not displaying in the headset. The errors at the bottom indicate HDR being enabled as well.
> 
> ```
> [2022-01-24 22:43:43.595] [REFramework] [info] [D3D12] Back buffer format is 24
> ```
> 
> 24 is the format for the HDR color format.
> 
> I don't have an HDR monitor, but I think there's a way to turn it off globally.
> 
> https://i.imgur.com/DnrOJuE.png

**Comment by iiStryker** [comment]
I'm using a TCL 50" S525 4K 60Hz HDR tv as my monitor. I turned off HDMI 2.1 forcing 1.4 and changed the color/pixel format to 8 bpc RGB 444 PC standard full rgb and that did the trick.

Have to load game in steam vr, remove the quest 2, cycle through to my game save and load into the game world ion pancake, put quest 2 back on and then it hooks.

It may have been switching the HDR HDMI compatibility off in my tv menu which was likely the differentiator. The GPU driver may have auto detected a 2.1 connection and enabled HDR by default?

I was able to leave my HDMI cable plugged in and got MHR to hook. Need to see if this helps RE7 & 8 hook without needing to disconnect the HDMI.  RE2 was flawless but I realize that was optimized and each games file structure is different

I love VR and the work these modders have done is absolutely amazing. thank you!
